src/controllers/AdminController.java

```diff
@@ -0,0 +1,35 @@
+package controllers;
+
+import config.AppException;
+import config.AppMessage;
+import org.springframework.stereotype.Component;
+import services.AdminService;
+
+@Component
+public class AdminController {
+    private AdminService adminService;
+    private AppMessage appMessage;
+
+    public AdminController(AdminService adminService, AppMessage appMessage) {
+        this.adminService = adminService;
+        this.appMessage = appMessage;
+    }
+
+    public String createDebitAccount(String username) throws AppException {
+        return this.adminService.addDebitAccount(username);
+    }
+
+    public String createCreditAccount(String username) {
+        return this.adminService.addCreditAccount(username) ?

```

Comment:
you can have one method for createDebitAccount and createCreditAccount named something like "createAccount". Maybe use enum to decide which type of account is created